pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -363,25 +367,25 @@
     ConsumerBuilder<T> receiverQueueSize(int receiverQueueSize);
 
     /**
-     * Group the consumer acknowledgments for the specified time.
+     * Sets amount of time for group consumer acknowledgments.
      *
-     * <p>By default, the consumer uses a 100 ms grouping time to send out the acknowledgments to the broker.
+     * <p>By default, the consumer uses a 100 ms grouping time to send out acknowledgments to the broker.
      *
-     * <p>Setting a group time of 0 sends out the acknowledgments immediately. A longer ack group time
-     * is more efficient at the expense of a slight increase in message re-deliveries after a failure.
+     * <p>Setting a group time of 0 sends out acknowledgments immediately. A longer acknowledgment group time
+     * is more efficient, but at the expense of a slight increase in message re-deliveries after a failure.
      *
      * @param delay
-     *            the max amount of time an acknowledgemnt can be delayed
+     *            the max amount of time an acknowledgement can be delayed
      * @param unit
      *            the time unit for the delay
      * @return the consumer builder instance
      */
     ConsumerBuilder<T> acknowledgmentGroupTime(long delay, TimeUnit unit);
 
     /**
-     * Group the consumer acknowledgments for the max size.
+     * Sets number of messages for group consumer acknowledgments.
```

Comment:
```suggestion
     * Set the number of messages for group consumer acknowledgments.
```
Is "group" necessary and accurate from tech perspective? From the grammar perspective, it should be "grouped" though.